### ISSUE TITLE
How to play all tracks from a directory in Library

### ISSUE THREAD

**Issue by chrisisbd** [orontee/argos#120]
Is there currently a way to play all tracks in a directory (as seen when Library browsing)?

This is how my mind works at least, I have my music arranged in a hierarchy of directories with the top level ones being genres and then, if you drill down, to sub-genres and/or artists.  So I select what I want to play by descending the hierarchy until I get to a directory with the tracks I want to play in it.

At present even the 'standard' ways of selecting more than one file don't work so it's not possible to select multiple files to play.  I can't find an 'add to playlist' either but maybe I'm missing something obvious (quite likely!).

Finally, on a vaguely related subject, what are 'albums' in argos?  Most of my directories do actually contain a single album but I don't see how this maps onto argos' concept of albums.

**Comment by orontee** [comment]
> Is there currently a way to play all tracks in a directory (as seen when Library browsing)?

No, there's no such capability. Support for Mopidy-File is quite young and lacks many obvious features.

> At present even the 'standard' ways of selecting more than one file don't work so it's not possible to select multiple files to play.

There's no support for multiple selection in directory views. 

> I can't find an 'add to playlist' either but maybe I'm missing something obvious (quite likely!).

This is already tracked here: https://github.com/orontee/argos/issues/117

> Finally, on a vaguely related subject, what are 'albums' in argos? Most of my directories do actually contain a single album but I don't see how this maps onto argos' concept of albums.

Argos doesn't introduce a concept of "album", it's just Mopidy's albums: Generated by Mopidy-Local when local files are indexed, or whatever Mopidy extension that support albums.
